docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+services:
+  n8n:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file: ".env"
+    volumes:
+      - ../n8n:/n8n
+      - ./:/test-workflows
+      - ./output:/output
+      - ./docker-data/dot-n8n:/home/node/.n8n
```

Comment:
I had to add this variable to make it work for me, without it it fails on git hooks which are disabled for docker     
```
environment:
      - DOCKER_BUILD=true 
```